tests/integration_tests/with_db/deployment/test_language_container_deployer_cli.py

```diff
@@ -41,63 +50,57 @@ def _call_deploy_language_container_deployer_cli(
         "--bucketfs-password", bucketfs_config.password,
         "--bucket", bucketfs_params.bucket,
         "--path-in-bucket", bucketfs_params.path_in_bucket,
-        "--container-file", container_path,
-        "--version", version,
         "--dsn", dsn,
         "--db-user", exasol_config.username,
         "--db-pass", exasol_config.password,
         "--language-alias", language_alias
     ]
+    if version is not None:
+        args_list += [
+            "--version", version,
+        ]
+    if container_path is not None:
+        args_list += [
+            "--container-file", container_path,
+        ]
     runner = CliRunner()
     result = runner.invoke(deploy.main, args_list)
-    assert result.exit_code == 0
-
-    # create a sample UDF using the new language alias
-    db_conn_test = pyexasol.connect(
-        dsn=dsn,
-        user=exasol_config.username,
-        password=exasol_config.password,
-        encryption=True,
-        websocket_sslopt={
-            "cert_reqs": ssl.CERT_NONE,
-        }
-    )
-    db_conn_test.execute(f"OPEN SCHEMA {schema}")
-    db_conn_test.execute(textwrap.dedent(f"""
-    CREATE OR REPLACE {language_alias} SCALAR SCRIPT "TEST_UDF"()
-    RETURNS BOOLEAN AS
+    return result
 
-    def run(ctx):
-        return True
 
-    /
-    """))
-    result = db_conn_test.execute('SELECT "TEST_UDF"()').fetchall()
-    return result
+def create_schema(pyexasol_connection: ExaConnection, schema: str):
+    pyexasol_connection.execute(f"DROP SCHEMA IF EXISTS {schema} CASCADE;")
+    pyexasol_connection.execute(f"CREATE SCHEMA IF NOT EXISTS {schema};")
```

Comment:
```suggestion
    pyexasol_connection.execute(f"CREATE SCHEMA {schema};")
```
Schema should not exist here

Reply:
it might be because we don't remove it at the moment

Reply:
The schema is dropped literally in the line above ;)